### ISSUE TITLE
Component property is set twice with the two-ways data-binding: first with an object, second with a string

### ISSUE THREAD

**Issue by sebastien-lamps** [tealpartners/ng-select2#12]
First, many thanks for this fabulous component

### Scenario:
In the [Demo with data changing demo](https://github.com/tealpartners/ng-select2/tree/master/src/app/demos/change), change 'value' property with this code:
```
  private _value: string;
  get value(): string {
    return this._value;
  }
  set value(value: string) {
    console.log('set value with ' + value);
    this._value = value;
  }
```
And select the value 'Cars 2'

### Expected:
   Console shows one message: 
`set value with car2`

### Actual:
   Console shows 2 messages:
```
set value with [object Object]
set value with car2
```

How to avoid this behavior ?

### PS: 
I know that the following lines are the cause of both messages:
ng-select2.component.ts:155
```
this.valueChanged.emit({
        value: newValue,
        data: this.element.select2('data'),
      });
this.propagateChange(newValue);
```

**Comment by icecoldfire** [comment]
Hey,

Could you please check that the issue is resolved in the new version 1.2.0?

Thank you for the issue report!

**Comment by sebastien-lamps** [comment]
Yes ! I think it works !
https://stackblitz.com/edit/ng-select2
Demos 'change' and 'multiple' show one change in console
Many thanks
